storage/common/src/test/scala/uk/ac/wellcome/platform/archive/common/fixtures/RandomThings.scala

```diff
@@ -12,9 +12,9 @@ trait RandomThings {
   def randomBagInfo =
     BagInfo(
       randomExternalIdentifier,
-      randomSourceOrganisation,
       randomPayloadOxum,
       randomLocalDate,
+      Some(randomSourceOrganisation),
```

Comment:
minor: use keyword arguments, so this is resilient to reordering?

Reply:
👍 Thanks